include/seqan3/alignment/pairwise/policy/unbanded_score_trace_dp_matrix_policy.hpp

```diff
@@ -0,0 +1,114 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \brief Provides seqan3::detail::unbanded_score_trace_dp_matrix.
+ * \author Rene Rahn <rene.rahn AT fu-berlin.de>
+ */
+
+#pragma once
+
+#include <range/v3/view/zip.hpp>
+
+#include <seqan3/alignment/matrix/trace_directions.hpp>
+#include <seqan3/alignment/pairwise/policy/unbanded_dp_matrix_policy.hpp>
+#include <seqan3/std/span.hpp>
+
+namespace seqan3::detail
+{
+/*!\brief Manages the allocation and provision of an unbanded dynamic programming matrix.
```

Comment:
what do you mean with `provision`? This is the exact same brief as `unbanded_dp_matrix_policy`... maybe add some difference like `that stores the trace and score information`..

Reply:
Again for me the dynamic programming matrix is a global property that might be represented internally in different vectors. I added a detail description.